Vic2ToHoI4/Source/Hoi4Outputter/Events/EventOptionOutputter.h

```diff
@@ -0,0 +1,18 @@
+#ifndef EVENT_OPTION_OUTPUTTER
```

Comment:
You can actually delete this file. EventOption already has a friend declaration and EventOption.cpp has the definition.

Reply:
ok, ignore I said this. After testing apparently having a friend function is not enough.